### DISCUSSION TITLE
Need some help about Student Pack

### DISCUSSION THREAD

**Discussion by 250king** [community/community#66743] [GitHub Education]
Now I have become a student in a university. I want to get the Student Pack in GitHub. But unlucky, our school doesn't issue the education email for us. So I wander if I can use the official document to verify my identify without education email.

**Comment by arafajs** [comment]
Yes, you can use official documents to verify your identity without an education email to get the GitHub Student Developer Pack. Here are the documents that you can use:

- A dated student ID
- A current registration document
- A time-stamped screenshot of your school portal
- A letter on school letterhead confirming you are a currently-registered student

You can upload these documents to GitHub when you apply for the Student Developer Pack. GitHub will review your documents and verify your identity. If your documents are approved, you will be granted access to the Student Developer Pack.

- Read this: https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/

**Reply by 250king** [reply]
But education email is still required(
![Screenshot_2023-09-12-20-36-46-71](https://github.com/community/community/assets/61256966/bd22505b-ea0b-4253-9d73-e66032ef75e6)
I don't know why Guangdong Police Academy isn't but Guangdong Communication Polytechnic is🤔

**Reply by tharcissentirandekura** [reply]
In that case, you can reach out to the school office to get the email setup but i am not sure it that is the best solution but in case!

**Reply by 250king** [reply]
Well, I have contacted with school network administration. The staff said the education email only provides to teachers. She can help me to ask for it. But I think it's impossible(
So I want to use official document instead of education email.

**Reply by 250king** [reply]
Actually, I want to contact GitHub Official to show the problem. But finally I can't find out any way to contact(

**Reply by gautam8404** [reply]
What worked for me was opening a ticket on GitHub support to change college information i.e. listing what problems i am facing

They took around a month to reply and it worked so maybe u can open a ticket on GitHub support?

**Reply by 250king** [reply]
How to do that?

**Comment by tharcissentirandekura** [comment]
I thin k you can upload any type of documents that shows that you are a student at that university. I think that will work and good luck

**Reply by 250king** [reply]
https://github.com/orgs/community/discussions/66743#discussioncomment-6978641
